OJP/OJP_Capabilities.xsd

```diff
@@ -0,0 +1,178 @@
+				<xs:sequence>
+					<xs:group ref="ServiceResponseContextGroup"/>
+					<xs:group ref="CapabilitiesResponseGroup"/>
+					<xs:element ref="siri:Extensions" minOccurs="0"/>
+				</xs:sequence>
+			</xs:extension>
+		</xs:complexContent>
+	</xs:complexType>
+	<xs:annotation>
+		<xs:documentation>=============================== Capabilities Response Group ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:group name="CapabilitiesResponseGroup">
+		<xs:annotation>
+			<xs:documentation>Capabilities response group.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="SystemDescription" type="xs:normalizedString" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>System description</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="CoveredArea" type="AreaStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>CoveredArea</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="CoveredModeList" type="ModeListOfEnumerations" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Covered list of modes</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="CoveredOperatorList" type="OperatorListOfEnumerations" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Covered list of operator names</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="ContainsRealTime" type="LimitationStatusEnumeration" default="false" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Limitations to realtime based on LimitationStatusEnumeration (NeTEx). Default is false.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="PrognosisWindowForward" type="xs:duration" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Prognosis window forward in time</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="PlanningHorizon" type="xs:duration" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Allowed planning horizon as xs:duration</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="ServiceList" type="ServiceListOfEnumerations" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>List of available OJP services.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="SustainsDisruptions" type="LimitationStatusEnumeration" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Whether this service contains disruption information.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="GeoSearchesPossible" type="xs:boolean" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Whether searches with coordinates are possible.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="OJPAlgorithmType" type="OJPAlgorithmTypeEnumeration" minOccurs="0">
```

Comment:
since this list is very specific for now (LinkingAlps + EU Spirit) shouldn't we have an open ended type to allow easy further extensions

Reply:
@Aurige : I think we will update this list with pull requests to master. And frankly I don't think that anybody else but this group should add new OJPAlgorithmTypes. Ok?